content/en/docs/setup/install/helm/index.md

```diff
@@ -166,6 +166,16 @@ installed above.
     $ kubectl delete namespace istio-system
     {{< /text >}}
 
+## Uninstall stable revision label resources
+
+If you decide to continue using the old control plane, instead of completing the upgrade,
```

Comment:
Should this say update instead of upgrade?

Reply:
Fixed